### ISSUE TITLE
2.1.1 install instructions for drbd

### ISSUE THREAD

**Issue by Ganeti-Issues-Migrator** [ganeti/ganeti#161]
Originally reported of Google Code with ID 105.

```
What software version are you running? Please provide the output of "gnt-
cluster --version" and "gnt-cluster version".

<b>What distribution are you using?</b>
gnt-cluster (ganeti) 2.1.1
software version 2.1.1
internode protocol: 30
configuration format: 201000
os api version: 15
export interface: 0

install instructions
(http://ganeti-doc.googlecode.com/svn/ganeti-2.1/html/install.html) -
section on DRBD, says to comment out resource definitions as follows:

skip resource r0 {
  ...
}

skip resource "r1" {
  ...
}

correct syntax is

skip {
resource r0 {
  ...
}
}

skip {
resource "r1" {
  ...
}
}

i.e., the entire "skipped" section has to be enclosed in {}s - otherwise
drbd throws an error during boot initialization


```
Originally added on 2010-05-02 11:05:08 +0000 UTC.

**Comment by Ganeti-Issues-Migrator** [comment]
```
-- Empty comment --
```
Originally added on 2010-07-28 23:41:08 +0000 UTC.

Changed State: Started

**Comment by Ganeti-Issues-Migrator** [comment]
```
Fixed, will be in the 2.2 documentation.

thanks!
```
Originally added on 2010-07-29 13:06:27 +0000 UTC.

Added Labels: Component-docs
Changed State: Fixed
Added to Milestone: Release2.2

**Comment by Ganeti-Issues-Migrator** [comment]
```
-- Empty comment --
```
Originally added on 2012-10-22 08:57:07 +0000 UTC.

Changed State: Released
